docs/specs/priority-queue.md

```diff
@@ -202,80 +214,13 @@ queue.Enqueue(userData, priority);
 
 To both priority queues:
 
-* If the `IComparer<T>` is not delivered, `Comparer<T>.Default` is summoned.
+* If the `IComparer<T>` is not delivered, `Comparer<T>.Default` is used.
```

Comment:
"delivered" is a weird word choice here. "Provided" would be better. 

Same comment for EqualityComparer